dotnetv3/cross-service/PhotoAssetManager/README.md

```diff
@@ -0,0 +1,106 @@
+
+![pam_overview.png](pam_overview.png)
+
+### Frontend
+The frontend is a single React app that uses the Cloudscape Design System. With the React app, users can authenticate with an Amazon Cognito flow. The app is deployed to an Amazon S3 bucket using the provided AWS CDK stacks and publicly exposed using an Amazon CloudFront distribution. The name is dynamically created during the AWS CDK deployment.
+
+![pam_ui.png](pam_ui.png)
+
+### Backend
+
+The backend of the PAM application is implemented with these AWS Lambda functions:
+
+- **Labels** - Serverless Api endpoint that returns the labels and label count of images in the S3 storage bucket.
+- **Upload** - Serverless Api endpoint that returns a presigned URL for uploading an image.
+- **DetectLabelsFunction** - S3 event function that is invoked when an image object is created in an S3 storage bucket. The new image is analyzed using AWS Rekognition, and label information is stored in a DynamoDB table.
+- **DownloadFunction** - API Gateway function that combines images into a zip file in an S3 storage bucket with a presigned URL for download, and sends a notification message using Amazon SNS.
+
+## ⚠️ Important
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Prerequisites
+
+To run the code in this example, you need the following:
+
++ An AWS account.
++ .NET 6
++ Docker Desktop
++ NodeJS 18+
++ To set up your development environment,
+  see [Setting up your AWS SDK for .NET environment](https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/net-dg-setup.html).
+
+## Create the resources
+
+Follow the instructions in the
+[README for the Photo Asset Manager application CDK](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/applications/photo-asset-manager/cdk/README.md).
+to use the AWS Cloud Development Kit (AWS CDK) or AWS Command Line Interface
+(AWS CLI) to create and manage the resources used in this example. You must be running Docker in order to complete the steps for this CDK.
+
+## Build the code
+
+The Visual Studio solution **PhotoAssetManager.sln** includes several projects with different purposes for this example.
+
+- **PamApi** - A Lambda Serverless Api project that includes an ApiController for the serverless endpoints, a Swagger UI for development environments, and the separate Download and DetectLabels functions. 
```

Comment:
- **PamApi** - A Lambda Serverless API project that includes an ApiController for the serverless endpoints, a Swagger UI for development environments, and separate Download and DetectLabels functions.